data_hub_api/docmaps/v2/api_input_typing.py

```diff
@@ -35,6 +35,15 @@
     }
 )
 
+ApiVorVersionsInput = TypedDict(
```

Comment:
minor: would rename to singular `ApiVorVersionInput` as it represents a single version